lib/src/components/NxSearchDropdown/NxSearchDropdown.scss

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-container-helpers';
+
+.nx-search-dropdown {
+  display: inline-block;
+  position: relative;
+
+  &__menu {
```

Comment:
Up until this point we have not used this syntax and I don't remember if there was a reason why we didn't. I'm curious as to why you decided to use it here.

Reply:
> > > I'm curious as to why you decided to use it here.
> > 
> > 
> > It aligned with my feelings of laziness that day. Up to you if you like it and want us to try to use it more going forward, or don't like it and want me to rip it out.
> 
> I'm not opposed to using this syntax going forward, so longs well all agree. @avnvbhatta @entroform ?

Works for me.

Reply:
+1 for me as well!